R/data.table.R

```diff
@@ -49,6 +49,32 @@ null.data.table = function() {
   setalloccol(ans)
 }
 
+rowwiseDT = function(..., key=NULL) {
+  x = substitute(list(...))[-1L]
```

Comment:
Just to satisfy my curiosity: why do you prefer `substitute(list(...))[-1L]` instead of `substitute(...())`? (The latter is faster, more elegant and with less typing.)

Reply:
I seen it more commonly used mailing lists

Reply:
Forgive my ignorance, I wasn't aware of this before. However, I can't find the doc of this trick in `?dots` or anywhere. In general, I prefer no tricks unless there're significant gains. 

Besides, other codes of data.table are using this style. So I'll let it be. If `substitute(...())` is preferred, then we should address all of the other codes in a separate PR, altogether.

<img width="765" alt="image" src="https://user-images.githubusercontent.com/8368933/76483945-5a673d80-6453-11ea-904b-86910adaeafa.png">

Reply:
No problem, and let it be the 'standard' way. When looking at the R source, I could not find any usage of `substitute(...())`, but  `substitute(list(...))[-1L]` occurs in many places. I will ask the gurus on r-devel whether `substitute(...())` is legal.